### DISCUSSION TITLE
Tiling Fiji/BigStitcher H5 format (development branch)

### DISCUSSION THREAD

**Discussion by drchrisch** [mesoSPIM/mesoSPIM-control#56] [Q&A]
I am not quite sure that I got the setup for tiling image acquisition and in particular the saving options (Fiji/BigStitcher H5 format) right. In particular, setting the correct file name to get a single hdf5 file is not clear to me from the CHANGELOG. Could I get a hint?

**Comment by nvladimus** [comment]
Hi! You don't have to set the H5 file name. By default, it will be 00000.h5, accompanied with 00000.xml file. You can add optional description when prompted, it will be added e.g. like `mouse_00000.h5`. The current software version lists 00000.h5 multiple times (one per each acquisition stack), but everything gets saved into a single file. It's just a bit confusing GUI at the moment!
